docs/aws.md

```diff
@@ -0,0 +1,90 @@
+# AWS
+
+To know how to price each resource it's good to check the [AWS Price Calculator](https://calculator.aws/#/estimate) and the CSV that we
+use for AWS has [this](https://docs.aws.amazon.com/cur/latest/userguide/product-columns.html) columns and format.
+
+## Adding new resources
+
+1. Familiarize yourself with the official AWS pricing page for the service as well as the Terraform documentation for the resource you want to add. Note all factors that influence the cost.
+2. Download and familiarize yourself with the pricing data CSV. This can be done by first checking the [index.json](https://pricing.us-east-1.amazonaws.com/offers/v1.0/aws/index.json), finding the respective service under the `offers` key and downloading the file at the URL under the `currentVersionUrl` (replace `json` with `csv`).
+3. Find the names of all columns that contain relevant cost factors and check that the `aws/field/field.go` file contains them - add them if this is not the case. The constant name should be a correct Go identifier, while the comment should contain the name as it appears in the CSV file.
```

Comment:
If adding a new field, the `ingester.go` might be missing ? https://github.com/cycloidio/terracost/blob/master/aws/ingester.go#L203